barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/mega_flavor.hpp

```diff
@@ -237,7 +233,10 @@ class MegaFlavor {
                 this->return_data_inverses,
             };
         }
-        auto get_to_be_shifted() { return DerivedEntities<DataType>::get_to_be_shifted(); }
+        auto get_to_be_shifted()
```

Comment:
hmmm was this broken before??

Reply:
not broken, we have this weird distinction between wire witnesses and derived witnesses.